build-support/bin/native/bootstrap.sh

```diff
@@ -16,24 +20,29 @@
 REPO_ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && cd ../../.. && pwd -P)
 source ${REPO_ROOT}/build-support/common.sh
 
-# Defines:
-# + RUST_OSX_MIN_VERSION: The minimum minor version of OSX supported by Rust; eg 7 for OSX 10.7.
-# + OSX_MAX_VERSION: The current latest OSX minor version; eg 12 for OSX Sierra 10.12.
-# + LIB_EXTENSION: The extension of native libraries.
-# + KERNEL: The lower-cased name of the kernel as reported by uname.
-# + OS_NAME: The name of the OS as seen by pants.
-# + OS_ID: The ID of the current OS as seen by pants.
-# Exposes:
-# + get_native_engine_version: Echoes the current native engine version.
-# + get_rust_osx_versions: Produces the osx minor versions supported by Rust one per line.
-# + get_rust_osx_ids: Produces the BinaryUtil osx os id paths supported by rust, one per line.
-# + get_rust_os_ids: Produces the BinaryUtil os id paths supported by rust, one per line.
-source ${REPO_ROOT}/build-support/bin/native/utils.sh
+# TODO(John Sirois): Eliminate this replication of BinaryUtil logic internal to pants code when
+# https://github.com/pantsbuild/pants/issues/4006 is complete.
```

Comment:
This issue is probably no longer relevant to this codepath.

Reply:
Agreed, killed. Also killed the emulation and just kept `LIB_EXTENSION`, which is the only variant bit needed any longer.